reco_utils/recommender/lightfm/lightfm_utils.py

```diff
@@ -0,0 +1,148 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import lightfm
+from lightfm.evaluation import precision_at_k, recall_at_k, auc_score
+
+def model_perf_plots(df):
+    """
+    Function to plot model performance metrics
+    Args:
+        df (Pandas dataframe): Dataframe in tidy format, with ['epoch','level','value'] columns
+    Returns:
+        matplotlib axes
+    """
```

Comment:
a minor comment for consistency with the rest of the library, we would do the comments like this:
```
    """Function to plot model performance metrics

    Args:
        df (pd.DataFrame): Dataframe in tidy format, with ['epoch','level','value'] columns

    Returns:
        obj: matplotlib axes
    """
```

Something that we (painfully) realised some time ago is that if we don't add a new line between Returns: and the previous comment, the docstrings are not parsed by sphinx, so we can't build the documentation